### PR TITLE
Data Queries with Tanstack and Axios

### PR DESCRIPTION
**Pull request by JonathanLadwig**
Styling leaves a lot to be desired, but it works.

### CODE REVIEW

File: manga-finder/src/pages/Error.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function Error() {
+  return (
+    <div className="error-page">
+      <div className="error-404 flex flex-col text-2xl">
+        <h2>Whoops! Error404</h2>
+        <img src="./src/assets/baka.jpeg" alt="baka!" />
```
Comment (MicNow27) [inline manga-finder/src/pages/Error.tsx]: The image doesn't appear for routes like `/manga/{manga-id}/foo` or `/browse/foo`.

Comment (JonathanLadwig) [reply manga-finder/src/pages/Error.tsx]: Placeholder for now, I'll try fix it though.

Comment (JonathanLadwig) [reply manga-finder/src/pages/Error.tsx]: Fixed, changed the src to "../src"

**Review by MicNow27 — COMMENTED** [review]
Card, line 26 and CardList, line 65, add a key attribute 
Screenshot 2023-03-20 at 15 31 04

File: manga-finder/src/components/TabsBar.tsx
Comment (MicNow27) [inline manga-finder/src/components/TabsBar.tsx:1]: This doesn't stick to the bottom of the viewport'; it goes to below the rendered page, i.e. user must scroll down to see it.

Comment (JonathanLadwig) [reply manga-finder/src/components/TabsBar.tsx:1]: Fixed in latest update. Working on map keys now too.

Comment (JonathanLadwig) [reply manga-finder/src/components/TabsBar.tsx:1]: Also fixed the map keys now too.

**Review by MicNow27 — COMMENTED** [review]
Getting this error when going to read a chapter 

Screenshot 2023-03-21 at 18 05 49

File: manga-finder/src/components/MangaReader.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  let counter = 1;
+  return (
+    <div className="manga-reader bg-dark w-screen h-auto flex flex-col items-center">
+      {chapterPages.map((page) => {
+        return (
+          <>
+            <img
+              src={
+                "https://uploads.mangadex.org/data/" + chapter.hash + "/" + page
```
Comment (MicNow27) [inline manga-finder/src/components/MangaReader.tsx:39]: Maybe try using the data-saver images to make loading quicker.

Comment (JonathanLadwig) [reply manga-finder/src/components/MangaReader.tsx:39]: Right, I'll do some last minute changes.

Comment (JonathanLadwig) [reply manga-finder/src/components/MangaReader.tsx:39]: Did the changes minus the data-saver, thought maybe people would like the quality despite the sometimes slow loading.

**Review by MicNow27 — APPROVED** [review]
Looks good

**Pull request merged**